internal/services/environment_settings/resources_environment_settings.go

```diff
@@ -118,6 +118,14 @@ func (r *EnvironmentSettingsResource) Schema(ctx context.Context, req resource.S
 									boolplanmodifier.UseStateForUnknown(),
 								},
 							},
+							"log_retention_period_in_days": schema.Int64Attribute{
+								Description:         "Is audit retention period v2",
+								MarkdownDescription: "Is audit retention period v2",
```

Comment:
you can remove the `Description` as the `MarkdownDescription` overrides it when added. You can remove all the Description from this resource

Reply:
OK I will remove all the descriptions.